java-generic-mqtt/src/main/java/com/cumulocity/generic/mqtt/client/model/GenericMqttMetadata.java

```diff
@@ -0,0 +1,66 @@
+package com.cumulocity.generic.mqtt.client.model;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.Map;
+
+/**
```

Comment:
nitpick: maybe say 'Represents the Metadata of a {@link GenericMqttMessage}.'

Reply:
Done.